mapie/aggregation_functions.py

```diff
@@ -112,8 +112,12 @@ def aggregate_all(agg_function: Optional[str], X: NDArray) -> NDArray:
     array([14.5, 14.5])
 
     """
-    if agg_function == "median":
-        return np.nanmedian(X, axis=1)
-    elif agg_function == "mean":
-        return np.nanmean(X, axis=1)
+    row_nan_mask = np.isnan(X).all(axis=1)
```

Comment:
If I understand correctly, the warning happens when the entire axis is NaN is that right?